modules/virt-enabling-usb-host-passthrough.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-usb-host-passthrough_{context}"]
+= Enabling USB host passthrough
+
+You can enable USB host passthrough at the cluster level.
+
+You specify a resource name and USB device name for each device you want first to add and then assign to a virtual machine (VM). You can allocate more than one device, each of which is known as a `selector` in the HyperConverged (HCO) custom resource (CR), to a single resource name. If you have multiple, identical USB devices on the cluster, you can choose to allocate a VM to a specific device.
```

Comment:
@xpivarc @tiraboschi The meaning of the word "selector" in particular can be lost in translation to other languages. Understood that that's what it's called in the yaml, but is there a descriptive adjective that could be added here to help make the meaning of the word less ambiguous? "selector option"? "selector parameter"?

Reply:
@stu-gott . @tiraboschi , @dbasunag, Thanks for all of your help on the issue of further qualifying selector. I've got Catherine Tomasko's okay to go ahead and get the PR merged as is.

(If we later find that there is a good additional descriptor for selector, you can let me know and I'll fix the content.) 

cc:@ctomasko